src/mame/jaleco/megasys1.cpp

```diff
@@ -1935,10 +2111,11 @@ void megasys1_state::system_D(machine_config &config)
 
 	GFXDECODE(config, m_gfxdecode, m_palette, gfx_abc);
 	PALETTE(config, m_palette, FUNC(megasys1_state::megasys1_palette)).set_format(palette_device::RGBx_555, 0x800/2);
-	MCFG_VIDEO_START_OVERRIDE(megasys1_state,megasys1)
 
 	MEGASYS1_TILEMAP(config, m_tmap[0], m_palette, 256*0);
+	m_tmap[0]->set_screen_tag("screen");
 	MEGASYS1_TILEMAP(config, m_tmap[1], m_palette, 256*1);
+	m_tmap[1]->set_screen_tag("screen");
```

Comment:
You’ve got a device finder for the screen. Why aren’t you using it in place of the literal tag here?